code/modules/projectiles/guns/projectile/shotgun.dm

```diff
@@ -65,7 +65,7 @@
 // RIOT SHOTGUN //
 
 /obj/item/weapon/gun/projectile/shotgun/riot //for spawn in the armory
-	name = "riot shotgun"
+	name = "\improper Washington 227 Shotgun"

```

Comment:
Shotgun should be lower case and I think it should continue to be called a riot shotgun